test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,73 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Test;
+
+public class UtilsTest {
+    @Test
+    public void isAnyNull() {
+        // empty list
+        assertFalse(Utils.isAnyNull());
+
+        // Any non-empty list
+        assertFalse(Utils.isAnyNull(new Object(), new Object()));
+        assertFalse(Utils.isAnyNull("test"));
+        assertFalse(Utils.isAnyNull(""));
+
+        // non empty list with just one at the beginning
+        assertTrue(Utils.isAnyNull((Object)null));
+        assertTrue(Utils.isAnyNull(null, "", new Object(), null));
+        assertTrue(Utils.isAnyNull(null, new Object(), new Object()));
+
+        // non empty list with nulls in the middle
+        assertTrue(Utils.isAnyNull(new Object(), null, null, "test"));
+        assertTrue(Utils.isAnyNull("", null, new Object()));
+
+        // non empty list with one null as the last element
+        assertTrue(Utils.isAnyNull("", new Object(), null));
+        assertTrue(Utils.isAnyNull(new Object(), new Object(), null));
+        
+        // test whether method peeks into the elements inside container
+        List<Object> nullList = createList((Object)null);
+        assertFalse(Utils.isAnyNull(nullList));
+    }
+
+    @Test
+    public void elementsAreUnique() throws Exception {
+        // empty list
+        assertAreUnique();
+
+        // only one object
+        assertAreUnique((Object)null);
+        assertAreUnique(1);
+        assertAreUnique("");
+        assertAreUnique("abc");
+
+        // all objects unique
+        assertAreUnique("abc", "ab", "a");
+        assertAreUnique(1, 2);
+
+        // some identical objects
+        assertNotUnique("abc", "", "abc", "ABC");
+        assertNotUnique(null, 1, new Integer(1));
+        assertNotUnique("", "abc", "a", "abc");
+        assertNotUnique(null, "a", "b", null);
```

Comment:
To make the tests clearer, maybe you could e.g., for this test `assertNotUnique(null, "a", "b", null)`, add a test above with `assertNotUnique(null, null)`, to make it clear that you are testing that `elementsAreUnique()` still works even if there are unique elements mixed in.

Also, it aids as a check against false negatives (e.g., to check that `elementsAreUnique(null, "a", "b", null)` return false because `elementsAreUnique(null, null)` return false and not because `elementsAreUnique(null, "a")` returns false).